### ISSUE TITLE
Transport ETH socket bind 

### ISSUE THREAD

**Issue by PeterGondek** [christoph2/pyxcp#132]
Hi All,

is it possible to add in transport/eth.py the possibility to bind the socket to concrete IP address defined in config?

For IPv6 when You have multiple IPs assigned to the ETH adapter, sometimes You need to pick up concrete IP.
[pyxcp_transport_eth_py.zip](https://github.com/christoph2/pyxcp/files/11622848/pyxcp_transport_eth_py.zip)

Best Regard,

Peter

**Comment by christoph2** [comment]
Yep, patch accepted, but I changed the names of the configuration parameters to `IPV6_BIND_TO_ADDRESS` resp. `IPV6_BIND_TO_PORT`.

I will integrate it in the main-line ASAP -- file `eth.py` is also affected by a comprehensive change not finished yet.
